services/distribution/src/main/java/app/coronawarn/server/services/distribution/objectstore/ObjectStoreAccess.java

```diff
@@ -101,7 +101,11 @@ public void putObject(LocalFile localFile, int maxAge) {
     Map<HeaderKey, String> headers = createHeaders(maxAge);
 
     logger.info("... uploading {}", s3Key);
-    this.client.putObject(bucket, s3Key, localFile.getFile(), headers);
+    try {
+      this.client.putObject(bucket, s3Key, localFile.getFile(), headers);
+    } catch (Exception e) {
+      logger.error("Can't upload file! ", e);
```

Comment:
Could you consider formulating it a bit more formal? "Cannot upload file" or such would fit a bit more into the tone we have in the other logs.